### DISCUSSION TITLE
How to retrieve the translated labels of an indicator field

### DISCUSSION THREAD

**Discussion by simbas** [OP-TED/eForms-SDK#369] [Q&A] [answered]
Hello everyone,

I am struggling to retrieve the translation for the indicator fields. For instance, for the field `BT-633-Organization` we are supposed to get the i18n labels based on the given keys `indicator|when-false|BT-633-Organization` and `indicator|when-true|BT-633-Organization`. But I don't find them anywhere in the translation files.

How to you populate the labels for those indicator radio fields?

Related information:
[Special label types documentation](https://docs.ted.europa.eu/eforms/latest/translations/index.html#_special_label_types)
[IndicatorInputField of the eforms-notice-editor implementation](https://github.com/OP-TED/eforms-notice-editor/blob/develop/src/main/resources/static/js/custom/notice-type-definition.js#L516)

**Comment by lizhong-zhang-IPD** [comment]
Hello, everyone,
I have the same problems of not finding the translations for indicator fields.
Is there anyone who knows the answers?
Thank you.

**Comment by eduardsmirnov** [comment]
Indicator fields have related codelist, e.g. "indicator_required.gc", you may get translations from that file (xml) or from translation file e.g. "code_en.xml" using constant "code|name|indicator.true". However they seem to be not translated and have translated values True and False.

**Comment by ferrakaop** [comment, marked as answer]
We are working on the labels for the technical codelists so for example code|name|indicator will have "Yes" as the label for true and "No" for false, and translated into the other 23 languages.

For now, we haven't defined any labels of type when-true or when-false as we have no immediate need for them when displaying the data, e.g. we can put "Natural Person: Yes".
